tests/performance/consumer/client.js

```diff
@@ -24,6 +26,13 @@ function getHeaders(odsCode = ODS_CODE) {
   };
 }
 
+function checkResponse(res) {
+  const is_success = check(res, { "status is 200": (r) => r.status === 200 });
+  if (!is_success) {
+    console.log(res.json());
```

Comment:
ran the tests, and seeing 10% issues (probs my env rather than this PR). The issues look like this in the console:
```
INFO[0002] {"resourceType":"OperationOutcome","issue":[{"severity":"error","code":"forbidden","details":{"coding":[{"system":"https://fhir.nhs.uk/ValueSet/Spine-ErrorOrWarningCode-1","code":"ACCESS DENIED","display":"Access has been denied to process this request"}]},"diagnostics":"The requested DocumentReference is not of a type that this organisation is allowed to access"}]}  source=console
```

Is it worth setting this to be `console.warn()` instead of `console.log()` to highlight that something unexpected has happened?